PowerUp/app/src/main/res/layout/pgame.xml

```diff
@@ -16,20 +17,47 @@
         android:layout_alignParentRight="true"
         android:layout_alignParentTop="true"
         android:background="@android:color/white"
-        android:backgroundTint="@color/abc_primary_text_disable_only_material_dark"
         android:ems="10"
         android:textColor="@android:color/black"
         android:textSize="@dimen/game_text_size" />
 
-    <ListView
-        android:id="@+id/mainListView"
+    <ImageView
+        android:id="@+id/askerImageView"
+        android:layout_width="@dimen/asker_image_view_width"
+        android:layout_height="@dimen/asker_image_view_height"
+        android:layout_alignParentRight="true"
+        android:layout_below="@id/scenarioNameEditText"
+        android:src="@drawable/marco" />
+
+    <LinearLayout
+
+        android:id="@+id/listviewlayout"
+
         android:layout_width="@dimen/main_list_view_width"
-        android:layout_height="@dimen/main_list_view_height"
+        android:layout_height="wrap_content"
         android:layout_alignParentRight="true"
-        android:layout_below="@+id/askerImageView"
-        android:background="@android:color/white"
-        android:backgroundTint="@color/abc_secondary_text_material_dark"
-        android:clickable="true"/>
+        android:layout_below="@id/askerImageView"
+        android:background="@android:color/black">
+        <powerup.systers.com.ListViewMaxHeight
+            android:id="@+id/mainListView"
+            android:layout_width="fill_parent"
+            android:layout_height="fill_parent"
+            app:maxHeight="@dimen/main_list_view_maxheight"
```

Comment:
This value is being hardcoded for 3 different device configurations, which is probably not what you want, since there are many different sizes, densities, orientations, etc. You may want to do this calculation programatically in the Java code instead so you calculate it from the height of the screen and the size the bottom bar takes. Right now, for instance, this is not working properly on a 5" device as it stays too short